### ISSUE TITLE
Transition of Modelica tickets from Trac to GitHub

### ISSUE THREAD

**Issue by beutlich** [beutlich/Modelica#1]
@modelica/MAP-LIB-write @dietmarw @tgoon @akiniwa @xuyangy @Root-nix @justnielsen @thorade @AtiyahElsheikh @urshanabi @danielthiel @afrhu @calbaker @lochel @mromanoni @hyumo @anicething @xie-dongping @fwt11 @Jeffbluz @caesarhao @sjoelund @MartinOtter @AHaumer @lvanfretti @tinrabuzin @raulrpearson @tbeu @rfranke @mtiller @wischhusen @d-hedberg @HansOlsson @hubertus65 @christiankral @casella @kristinmajetta @andreas-junghanns @DagBruck @ECClauss @GallLeo @ThomasBoedrich @ziske

Dear Modelica-repository contributors and watchers,

there will be many issues automatically generated during the scheduled transition of the MSL-related tickets from https://trac.modelica.org/Modelica to https://github.com/modelica/Modelica/issues. The transition will take place at one of the next weekends.

In order to avoid notification spam, it is recommended to ignore the notifications in the repository settings

![2017-01-09 15_03_48-modelica_modelica_ free standard conform library from the modelica association](https://cloud.githubusercontent.com/assets/14896695/21769118/6123fd32-d67d-11e6-9be6-6ff9e2a54e76.png)

or generally emails in the GitHub setttings.

![2017-01-09 15_04_27-notification settings](https://cloud.githubusercontent.com/assets/14896695/21769128/68ac53ba-d67d-11e6-9906-edfcb84ab8d5.png)
It also must be avoided to open new tickets or pull requests while the transition is running.

Once the transition is finished I will update this ticket, such that you will be notified once again.

Best regards,
Thomas Beutlich

**Comment by beutlich** [comment]
The transition of the 241 open and 1008 closed MSL-related tickets from [Trac](https://trac.modelica.org/Modelica) to [GitHub](https://github.com/modelica/Modelica) is scheduled for Friday, January 13th at 7pm UTC. It is expected to run for about 40 hours over the weekend.

Please adapt your GitHub notification settings to avoid any unsolicited notifications (on GitHub web account or by email). It is recommended to **not** watch the [MSL repository](https://github.com/modelica/Modelica) and to deactivate the email notifications for **Participating** at least.

Once the transition has succeeded I will update this ticket, such that you can reset your notification settings.

Thank you for your collaboration.

**Comment by beutlich** [comment]
The transistion of the Trac tickets to GitHub issues finished succesfully. You are now free to watch the [MSL repository](https://github.com/modelica/Modelica) again, clone the repository, open issues and prepare pull requests.

If you want to browse the closed issues, it is recommended to add "NOT Placeholder in:title" in the search filter in order to filter out the dummy issues.

**Comment by thorade** [comment]
Just to be 100% sure: Does this also mean the code development has moved from subversion to git? Everything is on github now?

**Comment by beutlich** [comment]
> Just to be 100% sure: Does this also mean the code development has moved from subversion to git? Everything is on github now?

Yes, MSL development is now on GitHub. The previous svn repo was set to r/o.
